packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -470,7 +513,7 @@ export class DebugConfigurationManager {
         // Between versions v1.26 and v1.27, the expected format of the data changed so that old stored data
         // may not contain the configuration key.
         if (data.current && 'configuration' in data.current) {
-            this.current = this.find(data.current.configuration, data.current.workspaceFolderUri, data.current.providerType);
+            this.current = this.find(data.current.configuration!, data.current.workspaceFolderUri, data.current.providerType);
         }
```

Comment:
Minor: When checking `data.current.configuration` instead of `'configuration' in data.current`, you can remove the non-null assertion again.

Reply:
Great suggestion!